features/step_eight.feature

```diff
@@ -59,7 +59,7 @@ Feature: Step eight page
   Scenario: No income
     Given I am a single person on step eight
     When I select no income
-    Then I am taken to step eleven
+    Then I am taken to step '11''probate'
```

Comment:
Hey @leoapost, it needs the quotes to pick up the step variables:

Then(/^I am taken to step '(\d+)''([^\"]*)'$/) do |step, url|
expect(current_path).to eq '/questions/fee'		 
expect(common_page.content.p.text).to have_content step
expect(current_path).to eq '/questions/' + url
end